docs/tutorials/join-dal-baker/run-dal-node.md

```diff
@@ -35,6 +39,9 @@ For example, this command initializes the DAL node with the address of a local `
    curl http://localhost:10732/p2p/points/info
    ```
 
+   where `10732` is the port on which the DAL node serves RPC calls.
+   The default port is 10732, but you can override it with the `--rpc-addr` argument.
```

Comment:
Do we have to mention 10732 twice in two sentences. Would it be relevant to merge it like:
```suggestion
   where `10732` is the default port on which the DAL node serves RPC calls.
   You can override it with the `--rpc-addr` argument.
```
?